how-to-guides/celestia-app-commands.md

```diff
@@ -156,7 +156,7 @@ proposals](https://docs.cosmos.network/v0.46/modules/gov/01_concepts.html#propos
 the list of [parameter defaults in the
 specs](https://github.com/celestiaorg/celestia-app/blob/0012451c4dc118767dd59bc8d341878b7a7cacdf/specs/src/specs/params.md),
 and the [x/paramfilter module
```

Comment:
```suggestion
and the [x/minfee module
```

@rootulp can you please advise on what the correct thing to link here is?

Reply:
minfee is a separate module. For now we can use a permalink to v3.x

https://github.com/celestiaorg/celestia-app/tree/1570b3d4937ac4418b3ecf77365d47c5f094ad8f/x/paramfilter